lib/tasks/allocations.rake

```diff
@@ -0,0 +1,6 @@
+namespace :allocations do
+  desc "Copy previous allocations data to allocation cycle year: rake allocations:copy_allocations_data_to_cycle_year"
+  task copy_allocations_data_to_cycle_year: :environment do
+    AllocationCopyDataService.call(allocation_cycle_year: Settings.allocation_cycle_year)
```

Comment:
I _think_ (correct me if I'm wrong @gpeng ) we'll need to run this task before updating the `allocation_cycle_year`, otherwise the dates will visually change before the data is in. It might be simplest for this rake task to accept `allocation_cycle_year ` as an argument.

Reply:
Yes that's right I think. If we pass it as an argument my `current` comment up there might be more important.

Reply:
Updated to allow you to pass in year.